packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -82,6 +84,7 @@ function MapsIndoorsMap(props) {
 		const categoryQueryParameter = queryStringParams.get('category');
 		const searchAllVenuesParameter = getBooleanQueryParameter(queryStringParams.get('searchAllVenues'));
 		const hideNonMatchesQueryParameter = getBooleanQueryParameter(queryStringParams.get('hideNonMatches'));
+        const showRoadNameLabelsQueryParameterBoolean = getBooleanQueryParameter(queryStringParams.get('showRoadNameLabels'));
```

Comment:
Some indentation issue here. Be a good scout and clean up (tabs vs spaces keeps being an issue in 2024 😂 ).

Reply:
Inside my editor everything is aligned (I tried tabs and spaces):

![Screenshot 2024-06-06 at 09 49 20](https://github.com/MapsPeople/web-ui/assets/56337591/006f5f51-04ec-4d74-a256-9ba08f381c3e)

Reply:
There are some mixed use of tabs and spaces here: https://github.com/MapsPeople/web-ui/blob/dfa65e7a89756a56c14785f099293ea88587076c/packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx#L84

Reply:
Done ✅